addons/website/static/src/components/media_dialog/image_selector.js

```diff
@@ -13,11 +13,39 @@ patch(ImageSelector.prototype, {
     }
 });
 
+patch(ImageSelector, {
+    mediaExtraClasses: ImageSelector.mediaExtraClasses.concat(
+        "social_media_img", 
+        (props) => {
+            if (
+                props.node
+                && props.node.classList.contains('fa') 
+                && props.node.closest('div')?.classList.contains('s_social_media')
+            ) {
+                return ('social_media_img');
+            };
+        },
+        (props) => {
+            if (
+                props.node
+                && props.node.classList.contains('fa') 
+                && props.node.closest('div')?.classList.contains('s_social_media')
+            ) {
+                for (const element of props.node.classList) {
+                    if (element.match(/fa-\d{1}x/)) {
+                        return element;
+                    }
+                }; 
```

Comment:
Instead of two functions, would not it be easier to support returning a list of classes? That way you'll avoid repeating the same condition twice.